tests/scripts/build-release.sh

```diff
@@ -18,12 +18,12 @@ if [[ ${BRANCH_NAME} =~ release- ]]; then
     git status &
     git diff &
     build/run make -C build/release publish BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=true AWS_ACCESS_KEY_ID=${AWS_USR} AWS_SECRET_ACCESS_KEY=${AWS_PSW} GIT_API_TOKEN=${GIT_API_TOKEN}
-elif [[ ${BRANCH_NAME} =~ master|tag ]]; then
+elif [[ ${BRANCH_NAME} == master ]]; then
```

Comment:
Seems like this still should apply to master and tagged releases

Reply:
added check for both master/tag and release.